tests/functional/reporting-auth.js

```diff
@@ -39,6 +39,68 @@ define(
             assert.include(text, "Report as"); //Report as FooUser (logged in)
           })
           .end();
+      },
+
+      "Show double feature - buglist shows on focus, hides on blur": function() {
+        return FunctionalHelpers.openPage(
+          this,
+          url("/issues/new"),
+          ".wc-ReportForm-actions-button"
+        )
+          .findById("url")
+          .click()
+          .type("https://test.")
+          .sleep(500)
+          .end()
+          .findById("description")
+          .click()
+          .end()
+          .sleep(9500)
+          .findById("bugList")
+          .getVisibleText()
+          .then(function(text) {
+            assert.equal(text, "");
+          });
+      },
+
+      "Show double feature - 3 issues with url found": function() {
+        return FunctionalHelpers.openPage(
+          this,
+          url("/issues/new"),
+          ".wc-ReportForm-actions-button"
+        )
+          .findById("url")
+          .click()
+          .type("https://webctest.")
+          .sleep(500)
+          .end()
+          .findById("bugList")
+          .findAllByTagName("li a")
+          .getVisibleText()
+          .then(function(texts) {
+            assert.isAtLeast(texts.length, 3, "found 3 issues");
```

Comment:
Do you expect there to be more than 3 issues? If not, I would recommend just using `assert.equal` if more than 3 is a meaningful failure.

(I don't know how many suggestions are shown yet)